script.js

```diff
@@ -1,4 +1,195 @@
+var WinCounter = 0;
+var LoseCounter = 0;
+var DrawCounter = 0;
+var TotalTurns = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myArray = ["scissors", "paper", "stone"];
+  var ComputerChoice = myArray[Math.floor(Math.random() * myArray.length)];
+  if (input == "scissors" && ComputerChoice == "stone") {
+    // Scissors lose to stone
+    LoseCounter = LoseCounter + 1;
+    TotalTurns = TotalTurns + 1;
+    return (myOutputValue =
+      "Computer chose " +
+      ComputerChoice +
+      "." +
+      "<br>" +
+      "You chose " +
+      input +
+      "." +
+      "<br>" +
+      "Sorry but you lose!" +
+      "<br>" +
+      "So far,you've been losing " +
+      LoseCounter +
+      "/" +
+      TotalTurns +
+      " turns.");
```

Comment:
Nice way of making the string changeable by putting ComputerChoice and input into the string instead of hardcoding. However this output is duplicated mulitple times, might wanna try refactoring it by using a function to output this string